### PR TITLE
Clarified eye tracking settings wording and added links to documentation

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Reworded the profile inspector settings to reflect their actual usage, and added a button directly in the inspector to bring users to our Eye Gaze setup documentation.

image

## Changes
- Fixes: #8390

### CODE REVIEW

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs [hunk 1]
```diff
@@ -82,7 +82,14 @@ public override void OnInspectorGUI()
                     EditorGUILayout.PropertyField(gazeCursorPrefab, GazeCursorPrefabContent);
                     EditorGUILayout.PropertyField(gazeProviderType);
                     EditorGUILayout.PropertyField(useHeadGazeOverride);
-                    EditorGUILayout.PropertyField(isEyeTrackingEnabled);
+
+                    EditorGUILayout.BeginHorizontal();
+                    EditorGUILayout.PropertyField(useEyeTrackingDataWhenAvailable, new GUIContent("Use Eye Tracking Data"));
+                    // Render a help link for getting started with eyetracking documentation
+                    string helpURL = "https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/input/eye-tracking/eye-tracking-basic-setup";
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs:89]:
```suggestion
                    string helpURL = "https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/features/input/eye-tracking/eye-tracking-basic-setup";
```

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs [hunk 2]
```diff
@@ -82,7 +82,14 @@ public override void OnInspectorGUI()
                     EditorGUILayout.PropertyField(gazeCursorPrefab, GazeCursorPrefabContent);
                     EditorGUILayout.PropertyField(gazeProviderType);
                     EditorGUILayout.PropertyField(useHeadGazeOverride);
-                    EditorGUILayout.PropertyField(isEyeTrackingEnabled);
+
+                    EditorGUILayout.BeginHorizontal();
+                    EditorGUILayout.PropertyField(useEyeTrackingDataWhenAvailable, new GUIContent("Use Eye Tracking Data"));
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs:87]: Could the "Use Eye Tracking Data" content be moved to the top of the class with the other `GUIContent`s?

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs (hunk 2 above)
Comment (RogPodge) [reply Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs]: Done

**Pull request merged**